File: programs/arrays-arrays_starting-game-2.cpp
#include "Conway.h"
#include <QApplication>
#include <QtWidgets>

int main(int argc, char* argv[]) {
  QApplication app(argc, argv);
  string title = "Conway's Game of Life";
  Conway game;
  QWidget window;
  window.setWindowTitle(QString::fromStdString(title));
  window.setFixedSize(game.grid->width(), game.grid->height());
  auto layout = new QVBoxLayout(&window);
  layout->addWidget(game.grid);
  window.show();
  game.mainloop();
  return app.exec();
}

File: programs/arrays-arrays_starting-game-2.java
public static void main(String[] args) {
    String title = "Conway's Game of Life";
    Conway game = new Conway();
    JFrame frame = new JFrame(title);
    frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
    frame.setResizable(false);
    frame.add(game.grid);
    frame.pack();
    frame.setVisible(true);
    game.mainloop();
}